src/handlers/privateResolver.ts

```diff
@@ -60,8 +74,11 @@ export const handler: AppSyncResolverHandler<
 
   switch (info.fieldName) {
     case 'registerLocalAuthority': {
+      registerLocalAuthoritySchema.parse(params);
```

Comment:
You can now do this const { VITE_AWS_REGION } = envSchema.parse(import.meta.env); and it removes the need for the line below. Just replace params as MutationRegisterLocalAuthorityArgs with the parse call.

Reply:
Thanks, this was a great tip!